ui/components/component-library/button-link/button-link.scss

```diff
@@ -11,7 +11,7 @@
     opacity: 0.5;
   }
 
-  &.mm-button--disabled {
+  &.mm-button-base--disabled {
```

Comment:
This is more of a question that will need further discussion but should we be adding a new class to handle these styles associated with this component rather that using a className from the child?